README.md

```diff
@@ -393,13 +390,31 @@ Make sure to include the following in your `AndroidManifest.xml` under the appli
 And that's it!
 
 ## Running on the emulator
+### Inside the AOSP tree
 
 After build, you can run the emulator in the terminal where you set up the environment variables with
 
 ```
 emulator
 ```
 
+### Outside the AOSP tree
+
+If you use a remote server to build your addon (It could be a good idea. Building the AOSP right now could be a really really hard job for some hardware) maybe you wanna extract the emulator to your personal environment. Actually, that's very easy and you already have all you need.
+
+Go to your Android Studio and create an emulator using the *Device Manager* tool. As we builded to a Automotive TARGET, select an hardware under the _Automotive_ category as well. For the _system image_, select (or download first if you don't have it yet) the API 34 image.
```

Comment:
```suggestion
Go to your Android Studio and create an emulator using the *Device Manager* tool. Since we builded to a Automotive TARGET, select an hardware under the _Automotive_ category as well. For the _system image_, select (or download first if you don't have it yet) the API 34 image.
```